packages/dx-react-grid-bootstrap3/src/templates/toolbar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const Toolbar = ({
+  children,
+  className,
+  style,
+  ...restProps
+}) => (
+  <div
+    className={classNames('panel-heading', className)}
+    style={{
+      display: 'flex',
+      alignItems: 'center',
+      position: 'relative',
+      minHeight: '55px',
```

Comment:
It doesn't fit well with the Darkly theme in the featured-theming demo.

Reply:
Looks [quite good](https://www.screencast.com/t/M2JWFetLei) on my side.

Reply:
The toolbar height is jumping when adding the first group.